searx/templates/oscar/preferences.html

```diff
@@ -27,9 +27,9 @@ <h5><a href="{{about.website}}" rel="noreferrer">{{about.website}}</a></h5>
     <span class="stacked-bar-chart-value">{{- stats[engine_name].time -}}</span>{{- "" -}}
     <span class="stacked-bar-chart" aria-labelledby="{{engine_name}}_chart" aria-hidden="true">
         {%- if max_rate95 is not none and max_rate95 > 0 -%}
-        <span style="width: calc(max(2px, 100%*{{ (stats[engine_name].time / max_rate95)|round(3) }}))" class="stacked-bar-chart-median"></span>{{- "" -}}
-        <span style="width: calc(100%*{{ ((stats[engine_name].rate80 - stats[engine_name].time) / max_rate95)|round(3) }})" class="stacked-bar-chart-rate80"></span>{{- "" -}}
-        <span style="width: calc(100%*{{ ((stats[engine_name].rate95 - stats[engine_name].rate80) / max_rate95)|round(3) }})" class="stacked-bar-chart-rate95"></span>{{- "" -}}
+        <div class="bar-chart-serie1 bar{{ (100 * (stats[engine_name].time / max_rate95))|round }}"></div>{{- "" -}}
```

Comment:
Why use `bar-chart-serie1` instead of `stacked-bar-chart-median` ?

In `/stats`, `bar-chart-serie1` is the HTTP time.
Here it is the median time.

Same question for the following lines:
* `bar-chart-serie2` / `stacked-bar-chart-rate80`
* `bar-chart-serie3` / `stacked-bar-chart-rate95`

In this PR:
![image](https://user-images.githubusercontent.com/1594191/132187636-26002180-699b-458a-b1a7-3dc4f4ac8f5f.png)

Using `stacked-bar-chart-*`:
![image](https://user-images.githubusercontent.com/1594191/132187830-c184fad9-e0a4-48bf-9d31-a78a7eb76fec.png)

UI choices:
* no color because the engine tab already contains a lot of data.
* highlight the median time (only lines for quartile 80 / 95)

Reply:
> Why use bar-chart-serie1 instead of stacked-bar-chart-median ?

My fail, no reason for .. fixed it to use stacked-bar-*